tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -412,5 +412,15 @@ RHODS Notification Drawer Should Contain
     [Documentation]    Verifies RHODS Notifications contains given Message
     [Arguments]     ${message}
     Click Element    xpath=//*[contains(@class,'notification-badge')]
-    Page Should Contain Element
-    ...    xpath=//*[contains(text(),'${message}')]
+    Wait Until Page Contains    text=${message}  timeout=2min
+    Click Element    xpath=//*[contains(@class,'notification-badge')]
+
+Clear RHODS Notifications
+    [Documentation]     Clears Notifications present in RHODS dashboard
+    Click Element    xpath=//*[contains(@class,'notification-badge')]
+    Sleep  20s
```

Comment:
why 20s waiting?

Reply:
After clicking on the notification icon , I tried to count the no.of notifications . But it was saying Element not interactable sometimes.

Reply:
Add msg for sleep?

Reply:
Updated.